custom_components/ha_kia_hyundai/vehicle_coordinator.py

```diff
@@ -363,3 +368,65 @@ def total_remaining_range_value(self) -> int:
             "lastVehicleInfo.vehicleStatusRpt.vehicleStatus.distanceToEmpty.value",
             int
         )
+
+    @property
+    def front_seat_options(self) -> dict:
+        """Return front seat options."""
+        return safely_get_json_value(
+            self.data,
+            "vehicleConfig.heatVentSeat.driverSeat",
+            dict,
+        )
+
+    @property
+    def rear_seat_options(self) -> dict:
+        """Return rear seat options."""
+        return safely_get_json_value(
+            self.data,
+            "vehicleConfig.heatVentSeat.rearLeftSeat",
+            dict,
+        )
+
+    @property
+    def heated_driver_seat(self) -> tuple:
+        """Get the status of the left front seat."""
+        return tuple(
+            safely_get_json_value(
+                self.data,
+                "lastVehicleInfo.vehicleStatusRpt.vehicleStatus.climate.heatVentSeat.driverSeat",
```

Comment:
on vehicles without heated seats this will throw an exception I think because heatVentSeat isn't defined so safely returns None

```
>>> tuple(None).values()
Traceback (most recent call last):
  File "<python-input-0>", line 1, in <module>
    tuple(None).values()
    ~~~~~^^^^^^
TypeError: 'NoneType' object is not iterable
```

see the sample responses in the api library for references but this is why using the explicit vehicleFeature.remoteFeature flags are needed

Reply:
Ok I see what you're saying, but I don't see what part of the code is calling that function if the seats aren't installed though.

Reply:
I see, standby